.sourcery.yml

```
@@ -0,0 +1,13 @@
+sources:
+  - ./Sources/Core
+templates:
+  include:
+    - ./Codegen/Templates
+  exclude:
+    - ./Codegen/Templates/Shared
+output:
+  path: ./Codegen/Generated
```

Code directory specificity?

```suggestion
  path: ./Codegen/Generated/FacebookCore
```

---

I guess it wouldn't hurt anything.